### PR TITLE
fix(css minify): add uppercase letters so that it works with css vars

### PR DESCRIPTION
**Pull request by obedparla**
Closes https://github.com/soflyy/unicorn/issues/870

Showcase : https://www.loom.com/share/cd6950bf58154ca786b711f9de7c191e

### CODE REVIEW

**Comment by louisreingold** [comment]
@FelipeMarcos @a-dubiel @alpharder @judereid you'll need to run `git submodule update` to get this change.

**Comment by felipemarcos** [comment]
What if the css var has a number?
Here is a suggestion: `.*:\s*;`


Example: --text-green-100: ;

File: src/CSS.php [hunk 1]
```diff
@@ -637,7 +637,7 @@ protected function stripEmptyTags($content)
      */
     protected function stripEmptyProperties($content)
     {
-        return preg_replace('/[a-z-:().#]*:\s*;/', '', $content);
+        return preg_replace('/[a-zA-Z-:().#]*:\s*;/', '', $content);
```
Comment (alpharder) [inline src/CSS.php:640]: Technically this should also support numbers and underscores: https://www.w3.org/TR/css-syntax-3/#ident-token-diagram

**Comment by obedparla** [comment]
> Example: --text-green-100: ;

@FelipeMarcos you're a genius and I hate you 😂. It doesn't work. I should've thought of it

image

I'll open another PR

**Comment by obedparla** [comment]
Fixed in: https://github.com/soflyy/minify/pull/3

**Pull request merged**